qubes-rpc/qubes.ShowInTerminal

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash --
+
+set -eu
+tmpdir="$(mktemp -d)"
+sock="${tmpdir//\"/\"\\\"\"/}/terminal.sock"
```

Comment:
This looks scary. And also is wrong - if path contains `"`, it will fail in the first socat call this way. `"` should not appear here anyway. IMO better only check and exit with an error if that happens, instead of trying to handle this situation fully.